static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -327,3 +393,14 @@ const StyledLink = styled(Link)`
   ${LinkStyles}
   color: ${p => p.theme.gray300};
 `;
+
+const CodecovWarning = styled('div')`
+  display: flex;
+  color: ${p => p.theme.errorText};
+  gap: ${space(0.75)};
+`;
+
+const WarningIcon = styled('div')`
```

Comment:
you shouldn't need this if you add align-items: center to CodecovWarning

Reply:
smart